src/snowflake/cli/api/output/types.py

```diff
@@ -27,6 +27,15 @@ def result(self):
         raise NotImplementedError()
 
 
+class EmptyResult(CommandResult):
+    def __init__(self):
+        pass
+
+    @property
+    def result(self):
+        return ""
```

Comment:
Any reason we don't want to return None (or just a bare return)? Does this have to be a string?

Reply:
How about just allowing commands to return `None`?

Reply:
@sfc-gh-fcampbell that's possible today:
https://github.com/snowflakedb/snowflake-cli/blob/4e50d2223352ef95581ab699b96bb90619cf663d/src/snowflake/cli/plugins/spcs/services/commands.py#L201-L203

We haven't had time to refactor this. For logs the solution is to have a `StreamResult`. 

I'm personally inclined to no return approach than having `return Empty`

Reply:
EmptyResult -> None makes sense, yes, happy to make that change if we think that a result should essentially be optional. I didn't go for it here because it's harder to forget then to return something (None being the default RV in Python and all). Not returning anything, to me, was an explicit choice that should be handled carefully for the reasons Tomasz pointed out. In this case, the stage diff command handles JSON formatting specifically, so I was OK with the empty/missing return.

Reply:
Ok, I made the change to None, since it's already a supported use case.